### ISSUE TITLE
A better name for the openpubkey/oidc package

### ISSUE THREAD

**Issue by EthanHeilman** [openpubkey/openpubkey#190]
There are too many oidc packages and similar dependencies: core-os/oidc, zitadel/oidc, core/go-oidc

What should we rename it to? For a while we were calling it simpleoidc, but that was too long as we reference the openpubkey/oidc package in so many places.

If someone wants to try their hand at the arcane art of discovering the true name of this package, I'm open to suggests. To avoid merge conflicts suggest the name here before creating the PR because I have a soon to be merged PR.

**Comment by RishavKumarSinha** [comment]
Hello there! I am interested in this issue, how about mykey/my-oidc as a simpler name?

**Comment by EthanHeilman** [comment]
Hi @RishavKumarSinha thanks for taking a look at this.
* We need to keep the module name as openpubkey, so it would be openpubkey/my-oidc.
* "my" isn't sufficiently meaningful. Who is the subject that "my" refers to? Maybe opk-oidc (opkoidc) or our-oidc? 

I've been thinking about "simple" synonyms that are shorter bare-oidc? Or maybe we rethink using oidc, maybe jwc (JSON Web Cryptography), jwsigs (JSON Web Sigs)?

What other name ideas you got?

**Comment by RishavKumarSinha** [comment]
Oh, thanks a lot for clarifying the requirement. I think the names "bare-oidc" or "jwc" suggested by you are already pretty great. The only other name, I can come up with at the moment is "jwauth" from JSON Web and Authorization.

**Comment by EthanHeilman** [comment]
Riffing off your "sigweb" idea, jwsig?

**Comment by RishavKumarSinha** [comment]
Indeed a good name combination, I like it.

**Comment by EthanHeilman** [comment]
@RishavKumarSinha Want to put a PR together that renames openpubkey/oidc to openpubkey/jwsig? 

Make sure to get rid of the simpleoidc package alias in https://github.com/openpubkey/openpubkey/blob/eba749e7d12b83798f2ab29ed684ba3bafed5e6f/providers/google.go#L32

**Comment by RishavKumarSinha** [comment]
Yes, of course, I would however need specific instructions to complete the PR request. How do I get started?

**Comment by EthanHeilman** [comment]
1. Fork OpenPubkey
2. A git branch on your fork for this PR
3. Get Openpubkey building and passing all tests on your machine `go ./...`.
4. Use refactoring tools to rename the package and all references to the package in OpenPubkey.
5. Ensure nothing is breaking after this change using `go ./...` and running some of the examples.
6. Git push to your branch
7. Open a PR into the openpubkey repo

**Comment by RishavKumarSinha** [comment]
@EthanHeilman, I have successfully managed to refactor the name. However, I am having difficulties with removing the alias 'simpleoidc'. I would appreciate it if you could help me out.

**Comment by EthanHeilman** [comment]
I see you have a fork. 
1. create a new branch `git branch <branchname>`
2. checkout that branch `git checkout <branchname>`
3. use git add to add the files you want to commit to that branch `git add <file1> <file2> ...`
4. commit those changes to that branch `git commit -m "your commit message here"`
5. push those changes to the remote `git push`


Then follow these instructions to create a PR.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

**Comment by RishavKumarSinha** [comment]
@EthanHeilman, I have successfully created the pull request, thanks a lot for taking the time to guide me through the process.

PR - https://github.com/openpubkey/openpubkey/pull/194
